lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<blockquote><s>Once the gamma has been computed, one usually considers the passing rate to have a final evaluation of the agreement between the reference and evaluated dose distributions. </s></blockquote><p> </p><p>I'm not quite sure what you're trying to say here, but my feeling is that it is best to remove this line. For common medical physics applications, the gamma index is well known to be a useful but still <em><u>insufficient </u></em>tool for comparison between dose distributions. </p><p> </p><blockquote>The passing rate is defined as the fraction of points having&nbsp;γ ≤ 1. <s>A threshold is chosen (e.g., 95%) and the test is passed if the passing rate exceeds such threshold.</s></blockquote><p> </p><p>Not sure there's a need to comment on a threshold/tolerance pass rate. It's not part of the API and is application-specific.</p><p> </p><blockquote><s>We check if in this case the test is passed (probably you already know the answer looking at the histogram above)</s></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
> > Once the gamma has been computed, one usually considers the passing rate to have a final evaluation of the agreement between the reference and evaluated dose distributions.
> 
> I'm not quite sure what you're trying to say here, but my feeling is that it is best to remove this line. For common medical physics applications, the gamma index is well known to be a useful but still _insufficient_ tool for comparison between dose distributions.

Here I'm trying to move to the "topic" of gamma pass rate, as the focus of the tute should be on how to speed up the gamma pass rate computation (where there is no need for an exact gamma index distribution). Maybe I could point out that the gamma index is still an insufficient tool for comparison between dose distributions, as you've mentioned.

> Not sure there's a need to comment on a threshold/tolerance pass rate. It's not part of the API and is application-specific.

Yes, I agree it's application-specific, but I was just trying to give an example on how the pass rate can be used. I think it might be useful to someone who's not familiar with this.

Reply:
<p>I understand. My first point is centred on the specific wording: "<em>considers the passing rate to have a </em><strong><em>final </em></strong><em>evaluation</em>". To me, this implies (admittedly softly) that a pass rate is a sufficient metric for all similar dose comparison tasks. I'm not suggesting that we do not cover the topic of gamma pass rate in the tutorial, I'm suggesting that we make no comment about the <em>utility </em>of the gamma pass rate at all.</p><p> </p><p>Similarly, for the second paragraph, I'm just trying to avoid any specific wording that implies that a 95% pass rate is some objective, general/well-accepted value. I'd prefer if we didn't even have a faint hint of opinion on this in PyMedPhys, and instead leave such decisions entirely to the user.</p><p> </p><p>Maybe something like the following could replace the text in the cell?</p><p> </p><blockquote>Once one has computed the gamma index distribution, one can compute a gamma pass rate: the fraction of data elements for which γ ≤ 1.</blockquote><blockquote>Suppose that, for our purposes, we consider the comparison to have "passed the test" if the gamma pass rate is above <strong>95%</strong> Let's check if the test passes:</blockquote><p> </p><p>It's a small point that I won't push. Just trying to explain the reasoning for my comments. :)</p>

Reply:
I agree, I'll be more vague about this. Let me know if the new phrasing works better